tracer/test/Datadog.Trace.Tests/Util/TestApiRequestFactory.cs

```diff
@@ -0,0 +1,141 @@
+﻿// <copyright file="TestApiRequestFactory.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Tests.Util;
+
+internal class TestApiRequestFactory : IApiRequestFactory
```

Comment:
We can delete this now, we have already added `TestApiRequest`, `TestRequestFactory` etc in a different PR

Reply:
Removed again in https://github.com/DataDog/dd-trace-dotnet/pull/2800/commits/05768f115c36416c2b5a4c20b89f57e8eee87016